### PR TITLE
Chore/documentation updates

### PR DESCRIPTION
**Pull request by goodrum-harness**

### CODE REVIEW

File: modules/secrets/text/README.md [hunk 1]
```diff
@@ -9,22 +9,57 @@ This module handle the creation and managment of Text Secrets by leveraging the
...
+
+provider "harness" {
+  endpoint         = var.harness_platform_url
+  account_id       = var.harness_platform_account
+  platform_api_key = var.harness_platform_key
+}
+
+```
+
+### Terraform required providers declaration
```
Comment (rssnyder) [inline modules/secrets/text/README.md:48]:
```suggestion
```

### Example setup of the Harness Provider with environment variables

You can also set up authentication with Harness through environment variables. To do this set the following items in your environment:

- HARNESS_ENDPOINT: Harness Platform URL, defaults to Harness SaaS URL: https://app.harness.io/gateway
- HARNESS_ACCOUNT_ID: Harness Platform Account Number
- HARNESS_PLATFORM_API_KEY: Harness Platform API Key for your account

### Terraform required providers declaration
```

how about something like this?

File: modules/secrets/text/README.md [hunk 2]
```diff
@@ -9,22 +9,57 @@ This module handle the creation and managment of Text Secrets by leveraging the
...
+}
+
+provider "harness" {
+  endpoint         = var.harness_platform_url
+  account_id       = var.harness_platform_account
+  platform_api_key = var.harness_platform_key
+}
+
+```
+
```
Comment (rssnyder) [inline modules/secrets/text/README.md:53]:
```suggestion

### Example setup of the Harness Provider with environment variables

You can also set up authentication with Harness through environment variables. To do this set the following items in your environment:

- HARNESS_ENDPOINT: Harness Platform URL, defaults to Harness SaaS URL: https://app.harness.io/gateway
- HARNESS_ACCOUNT_ID: Harness Platform Account Number
- HARNESS_PLATFORM_API_KEY: Harness Platform API Key for your account

```

Comment (rssnyder) [reply modules/secrets/text/README.md:53]: thoughts on adding this to better showcase setting these via env?

Comment (goodrum-harness) [reply modules/secrets/text/README.md:53]: Makes sense to me.  I pushed the update to the documentation.

**Review by rssnyder — APPROVED** [review]

**Pull request merged**